frontend/src/features/permits/helpers/payment.ts

```diff
@@ -24,9 +28,9 @@ export const getMotiPaymentDetails = (params: URLSearchParams): MotiPaymentDetai
     avsProcessed: getDefaultRequiredVal("", params.get("avsProcessed")),
     avsResult: getDefaultRequiredVal("", params.get("avsResult")),
     cardType: getDefaultRequiredVal(CARD_TYPES.VI, params.get("cardType")) as CardType,
-    cvdId: applyWhenNotNullable((cvdId) => Number(cvdId), params.get("cvdId"), 0),
-    trnApproved: applyWhenNotNullable((trnApproved) => Number(trnApproved), params.get("trnApproved"), 0),
-    messageId: getDefaultRequiredVal("", params.get("messageId")),
+    cvdId: 1, // applyWhenNotNullable((cvdId) => Number(cvdId), params.get("cvdId"), 0),
+    trnApproved: trnApproved,
+    messageId: '1', // getDefaultRequiredVal("", params.get("messageId")),
```

Comment:
needed to hardcode both `cvdId` and `messageId` values in order for submissions to work, as they weren't showing up properly in the response (from PayBC?) that we're parsing here. Not sure if these are just "extra" values no longer needed going from MOTI to PayBC